src/ray/raylet/node_manager.cc

```diff
@@ -1765,79 +1782,131 @@ bool NodeManager::AssignTask(const Task &task) {
     worker->SetTaskResourceIds(acquired_resources);
   }
 
+  auto task_id = spec.TaskId();
+  auto finish_assign_task_callback = [this, worker, task_id](Status status) {
```

Comment:
Code in this callback didn't change, right?

Reply:
sure, that should look simpler. thanks.